src/actions/index.js

```diff
@@ -1,8 +1,20 @@
 // we'll need axios
-
+import axios from "axios";
 // we'll need to create 3 different action types here.
 // one for fetching, one for success and one for failure
-
+export const FETCHING_DATA = "FETCHING_DATA";
+export const FETCHING_SUCCESS = "FETCHING_SUCCESS";
+export const FETCHING_ERROR = "FETCHING_ERROR";
 // our action creator will be a function that returns a function
 // the url to fetch characters from is `https://swapi.co/api/people/`
 // remember that now we have controll over our thunk-based action creator
+export const getDATA = () => dispatch => {
```

Comment:
I'd make your action creator name a bit less general, say `getStarwarsChars` instead of `getData`